Blackstar/ContentChunk.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Linq;
+
+namespace Blackstar
+{
+    public class ContentChunk
```

Comment:
Hrmmm splitting this into separate files destroyed the diff; sorry... All I changed here was to add the `Content` prefix.